src/components/Spinner/README.md

```diff
@@ -0,0 +1,12 @@
+# Spinner
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Spinner/story.js)
+
+## Props
+
+| Name          | Type      | Default   | Required   | Description                                          |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `children`     | `React.ReactNode` |         | -          | If the spinner has children, it will show an opaque overlay over the children when it is spinning |
+| `spinning`     | `boolean`         | true    | -          | If passed, it will control whether the spinner is shown or not |
+| `emphasized`   | `boolean`         | false   | -          | If true, it will render a large spinner |
```

Comment:
We should try to keep props similar or intuitive. So instead of emphasised, we can provide a `size` prop which can have some levels `sm | md | lg`, or a `number`.

Reply:
...or `large` (boolean, default false)?

Reply:
@iamsolankiamit @grahammcculloch The reason I went with `emphasized` is we used that terminology in other components (`Input`, `Select`, and `ProgressBar` so far), so it is just for consistency. About the multiple values, that's not something we want to support. We want to be very opinionated and not allow users to specify whatever value they want, so a boolean works better. We might add it to the theme if necessary, but I would rather do it only when we see a real need for it, and not just for the sake of adding it.